src/vm.rs

```diff
@@ -131,6 +131,34 @@ impl core::fmt::Debug for Delegate {
     }
 }
 
+#[cfg(feature = "variable-lookbehinds")]
+#[derive(Clone)]
+/// Delegate matching in reverse to regex-automata
+pub struct ReverseSearch {
```

Comment:
Thinking about it, I'm not sure I like this naming. It is essentially a Delegate matched backwards/in reverse. Nothing to do with searching 🙂